src/loader.js

```diff
@@ -0,0 +1,91 @@
+let assume = require('assume');
+let debug = require('debug')('taskcluster:loader');
+let Promise = require('promise');
+
+function loader(componentDirectory, requiredVirtualComponents) {
+  /**
```

Comment:
this isn't a doc comment, aren't those usuall before the function they document.